client/components/payments-activity/payments-activity-data.tsx

```diff
@@ -10,17 +11,43 @@ import HelpOutlineIcon from 'gridicons/dist/help-outline';
  */
 import PaymentsDataTile from './payments-data-tile';
 import { ClickTooltip } from '../tooltip';
+import { usePaymentActivityData } from 'wcpay/data';
 import { getAdminUrl } from 'wcpay/utils';
+import { DateRange, PaymentsActivityData } from './types';
```

Comment:
This import is no longer used and the name conflicts with the component defined on line 33.

```suggestion
import { DateRange } from './types';
```

Reply:
Good catch, resolved https://github.com/Automattic/woocommerce-payments/pull/8589/commits/e5bd64321a34e3283c08a0ee5f726ebbdcd8e41f